Assets/Shopify/Plugins/iOS/Shopify/ApplePayButtonGenerator.mm

```diff
@@ -54,29 +54,18 @@ PKPaymentButtonType SBPayButtonTypeFromString(const char *type) {
     
     // Generates a base64 encoded string containing the rendered image output of a PKPaymentButton from iOS.
     const char* _GenerateApplePayButtonImage(const char *type, const char* style, float width, 
-                                             float height, bool includeMinMargin,
-                                             bool includeMinimumSpacingForIOS) {
```

Comment:
The actual method doubled up on the `includeMinMargin` and `includeMinimumSpacingForiOS` parameters but the compiler never caught this! This resulted in undefined values being passed in which was causing a lot of headaches debugging this.